recipe/meta.yaml

```diff
@@ -1,21 +1,23 @@
-{% set version = "4.13.0" %}
+{% set name = "conda" %}
+{% set version = "4.14.0" %}
+{% set build_number = "0" %}
+{% set sha256 = "9b9fa3e20a2bcdd30496fc886bdd28b102e3a87eb3cdb4659b0a7e4e9edd52a8" %}
 # Running the upstream test suite requires the inclusion of test files, which
 # balloons the size of the package and occasionally triggers false-positive
 # security warnings; values can be "yes" or "no".
 {% set run_upstream_tests = "no" %}
 
 package:
-  name: conda
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: conda-{{ version }}.tar.gz
-  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
-  sha256: 30c4cee48ef3acc4e2d802c05baee44f5385f55b0c23c547b0961c51e1d140d2
+  url: https://github.com/conda/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
   skip: True  # [py<36]
```

Comment:
We should skip `py<37`, see https://github.com/conda/conda/blob/20ea9d5a7a658ab95acd2abcd5b7484f11686110/setup.py#L87